rllib/connectors/tests/test_agent.py

```diff
@@ -430,10 +430,9 @@ def test_vr_connector_with_multiple_buffers(self):
             # next state and action at time t-1 are the following
             timestep_data = {
                 SampleBatch.NEXT_OBS: obs_arrs[t],
-                SampleBatch.ACTIONS: (
-                    np.zeros_like(act_arrs[0]) if t == 0 else act_arrs[t - 1]
-                ),
             }
+            if t > 0:
```

Comment:
This is just to make it clear that this will not be used on the "0th" timestep anyway. The first timeste_data is only the initial obs that initializes the agent collector inside ViewRequirementsConnector.